packages/oak/lib/index.stories.js

```diff
@@ -214,6 +39,15 @@ export const basicConfig = () => {
             type: 'richtext',
           }],
         }],
+      } : editor === 'prosemirror' ? {
+        overrides: [{
+          type: 'component',
+          components: ['text', 'title', 'button'],
+          fields: [{
+            key: 'content',
+            type: 'prosemirror',
```

Comment:
prose mirror plugin type should still be richtext, and should replace the default richtext plugin as-is